app/services/affiliate-slots.js

```diff
@@ -95,6 +96,10 @@ export default Service.extend({
   currentVertical: readOnly('wikiVariables.vertical'),
   currentCountry: readOnly('geo.country'),
 
+  isLaunched: computed('wikiVariables.affiliateUnitEnabledDate', function () {
```

Comment:
Should this be "private"?

Reply:
I was just mirroring what we already heave for existing computed properties /shrug